ui/pages/swaps/awaiting-signatures/swap-step-icon.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default function SwapStepIcon({ stepNumber = 1 }) {
```

Comment:
Should I rename it to `StepIcon` and move it to `ui/components/ui`, so it can be reused in other features?

Reply:
It's for these icons (1 and 2):
![image](https://user-images.githubusercontent.com/80175477/117073178-67de2780-ace6-11eb-99ab-e33295d40ced.png)

Reply:
in a new PR! Would welcome that :D